tests/integration_tests.rs

```diff
@@ -1499,6 +1499,80 @@ fn ignored_suffix_arg() {
         .stderr("");
 }
 
+#[test]
+fn no_line_wrapping_when_set_to_never() {
+    let expected = "abcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyz
+";
+
+    bat()
+        .arg("--style=rule")
+        .arg("--color=never")
+        .arg("--decorations=always")
+        .arg("--wrap=never")
+        .arg("--terminal-width=80")
+        .arg("80-columns.txt")
+        .assert()
+        .success()
+        .stdout(expected)
+        .stderr("");
+}
+
+#[test]
+fn line_wrapping_when_auto() {
+    let expected =
+        "abcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcde
+fghigklmnopqrstuvxyz
+";
+
+    bat()
+        .arg("--color=never")
+        .arg("--style=rule")
+        .arg("--decorations=always")
+        .arg("--wrap=auto")
+        .arg("--terminal-width=80")
+        .arg("80-columns.txt")
+        .assert()
+        .success()
+        .stdout(expected)
+        .stderr("");
+}
+
+#[test]
+fn no_line_wrapping_with_s_flag() {
+    let expected =
+        "abcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyz\n";
+
+    bat()
+        .arg("--color=never")
+        .arg("--style=rule")
+        .arg("--decorations=always")
+        .arg("-S")
+        .arg("--terminal-width=80")
+        .arg("80-columns.txt")
+        .assert()
+        .success()
+        .stdout(expected)
+        .stderr("");
+}
+
+#[test]
+fn no_wrapping_with_chop_long_lines() {
+    let expected =
+        "abcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyz\n";
+
+    bat()
```

Comment:
There is a substantial amount of copy-paste here. Would be nice to introduce a helper function that takes as parameter the only thing that varies among the tests (expected output and the wrapping arg).

Reply:
Added a helper function.

Reply:
I added the helper function in the `tests/integration_tests.rs` file, but is this the correct procedure? I don't see any other helper functions in this file, so I figured I should ask.

Reply:
Yep it was correct to put the helper next to the functions that need it

I don't expect any other code to ever need that helper, so it makes sense to put it where you put it